### PR TITLE
Adding telekrischi coordinates

### PR DESCRIPTION
**Pull request by telekrischi**
@githubteacher What do you think

### CODE REVIEW

**Comment by telekrischi** [comment]
Everything's fine so far.

**Comment by Nawel17** [comment]
@telekrischi thislooks great! 🎉

File: _pins/telekrischi.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: YOUR-USERNAME
+latitude: 48.135125
+longitude: 11.58198
```
Comment (antarmukhee) [inline _pins/telekrischi.json]: Cool location !

**Comment by antarmukhee** [comment]
well done!

File: _pins/telekrischi.json (hunk 1 above)
Comment (Pav2712) [reply _pins/telekrischi.json]: Did your PIN appear on the map or does it take some time ?  I'm not seeing mine yet

**Pull request merged**